package.json

```diff
@@ -1,29 +1,19 @@
 {
-  "name": "Rambo-AuthorsHaven",
+  "name": "rambo-authors-haven",
```

Comment:
why this?

Reply:
eslint was flagging it as an error will send you the screenshot